### PR TITLE
fix(YouTube): fix community post author

### LINKED ISSUE #5576: Youtube | youtube.com/channel/XXX/community?lb=XXX
### Description

The YouTube presence does not fetch the author of a community post correctly. This issue is caused by the following line:
https://github.com/PreMiD/Presences/blob/486723e5e68470938ff52ba9a7d746eb8353d4cf/websites/Y/YouTube/presence.ts#L377
And the correct selector for community posts is `document.querySelector("#author-text > span").textContent`.
The reason I'm not making a pull request is that I don't know in what context the above selector is valid, so I am opening this so it can be investigated, possibly by the presence author/contributors

### Steps to Reproduce

1. Open a community post
2. Observe how the channel name in your presence is displayed as "null"

### Screenshots

image

### Expected Behavior

The presence fetches the name of the community post author correctly

### Actual Behavior

The presence shows the name of the author as null

### Page URL

https://www.youtube.com/channel/UCWLjpFuoxoFVhgKYQEgu84w/community?lb=UgkxmdJGAmku6W-LuRlTd_tA_TDogNGl8iyA

### Operating System (OS)

Windows 11

### Browser

Google Chrome

### Browser Version

97.0.4692.71

### PR DESCRIPTION
**Pull request by down-bad**
**Describe the changes in this pull request:**

Resolves #5576 
Also fixes channel pic not showing in some instances

image

### CODE REVIEW

File: websites/Y/YouTube/presence.ts [hunk 1]
```diff
@@ -424,11 +438,24 @@ presence.on("UpdateData", async () => {
...
+              "#avatar.ytd-c4-tabbed-header-renderer > img"
+            )
+            ?.src.replace("=s176", "=s512") ??
+          document //when viewing a community post
+            .querySelector<HTMLImageElement>(
+              "#author-thumbnail > a > yt-img-shadow > img"
+            )
+            ?.src.replace("=s76", "=s512") ??
+          "yt_lg";
+        if (channelImg !== "") presenceData.largeImageKey = channelImg;
```
Comment (EncryptedDev) [inline websites/Y/YouTube/presence.ts:458]: An empty string is falsey but test it first pelase 
```suggestion
        if (!channelImg) presenceData.largeImageKey = channelImg;
```

File: websites/Y/YouTube/dist/metadata.json [hunk 2]
```diff
@@ -120,7 +120,7 @@
       },
       "title": "Show Channel's Picture",
       "icon": "fas fa-user",
-      "value": false
+      "value": true
```
Comment (EncryptedDev) [inline websites/Y/YouTube/dist/metadata.json:123]: Why did you make this change?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/Y/YouTube/dist/metadata.json (hunk 2 above)
Comment (down-bad) [reply websites/Y/YouTube/dist/metadata.json:123]: oops, forgot it while i was testing it so i dont have to turn it on every time

File: websites/Y/YouTube/presence.ts (hunk 1 above)
Comment (down-bad) [reply websites/Y/YouTube/presence.ts:458]: this doesn't work

Comment (EncryptedDev) [reply websites/Y/YouTube/presence.ts:458]: I didn't read it properly, try this: 
```suggestion
        if (channelImg) presenceData.largeImageKey = channelImg;
```

Comment (down-bad) [reply websites/Y/YouTube/presence.ts:458]: Oh ye now it does

File: websites/Y/YouTube/presence.ts [hunk 3]
```diff
@@ -366,7 +366,21 @@ presence.on("UpdateData", async () => {
     ) {
       //Sometimes causes problems
       let user: string;
+      //get user title when viewing a community post
```
Comment (ImRodry) [inline websites/Y/YouTube/presence.ts:369]: can you add these comments to the other selectors if you can figure out what they are please 🙏🏻

Comment (down-bad) [reply websites/Y/YouTube/presence.ts:369]: Sure but I don't understand what the 3rd one does

Comment (ImRodry) [reply websites/Y/YouTube/presence.ts:369]: that's fine ig, do it for the ones you can

Comment (down-bad) [reply websites/Y/YouTube/presence.ts:369]: Actually nvm

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**